examples/meta/src/evaluation/cross_validation.sg

```diff
@@ -0,0 +1,94 @@
+#  Cross Validation for Binary Classifier
+CSVFile f_feats_train("../../data/classifier_binary_2d_linear_features_train.dat")
+CSVFile f_feats_test("../../data/classifier_binary_2d_linear_features_test.dat")
+CSVFile f_labels_train("../../data/classifier_binary_2d_linear_labels_train.dat")
+CSVFile f_labels_test("../../data/classifier_binary_2d_linear_labels_test.dat")
+Math:init_random(2)
+
+#![create_features]
+Features features_train = features(f_feats_train)
+BinaryLabels labels_train(f_labels_train)
+Features features_test = features(f_feats_test)
+BinaryLabels labels_test(f_labels_test)
+#![create_features]
+
+#OK
+#![set_parameters]
+real C = 1.0
+real epsilon = 0.001
+#![set_parameters]
+
+#OK
+#![create_instance]
+Machine svm = machine("LibLinear", labels=labels_train)
+svm.put("liblinear_solver_type", enum LIBLINEAR_SOLVER_TYPE.L2R_L2LOSS_SVC)
+svm.put("epsilon", epsilon)
+#![create_instance]
+
+#![create_cross_validation]
+StratifiedCrossValidationSplitting splitting_strategy(labels_train, 2)
+AccuracyMeasure evaluation_criterion()
```

Comment:
new API and factory pls, see other examples

Reply:
@karlnapf  Other examples in cross_validation use AccuracyMeasure evaluation_criterion(), do you mean another example?

Reply:
`evaluation_cirterion = evaluation("AccuracyMeasure")`

Reply:
What about this one, @FaroukY?